### ISSUE TITLE
Closing Script Tags Not Highlighting Correctly

### ISSUE THREAD

**Issue by grayghostvisuals** [atom/solarized-dark-syntax#69]
As you can see in the screenshot below the closing script tag is discolored and incorrectly highlighted.

![screenshot 2016-09-13 12 35 49](https://cloud.githubusercontent.com/assets/934322/18513689/87e9c50a-7a5c-11e6-874d-2df2496ec2b8.png)

**Comment by simurai** [comment]
Can't reproduce your example:

```html
<script src="//secure.leadforensics.com/js/79075.js"></script>
<noscript>&lt;img src="//secure.leadforensics.com/js/79075.js" style="display:none;"&gt;</noscript>
```

![screen shot 2016-09-20 at 11 30 44 am](https://cloud.githubusercontent.com/assets/378023/18655661/b38cdd8c-7f25-11e6-9f60-0d0d80ed0e62.png)

What grammar are you using, normal HTML? 

![screen shot 2016-09-20 at 11 32 57 am](https://cloud.githubusercontent.com/assets/378023/18655705/213fb4f8-7f26-11e6-9f0b-16da379cae22.png)

**Comment by grayghostvisuals** [comment]
@simurai Happens here in this context 

```html
<!doctype html>
<html lang="en">
<body>
  <head>
    <script src="//secure.leadforensics.com/js/79075.js"></script>
    <noscript>&lt;img src="//secure.leadforensics.com/js/79075.js" style="display:none;"&gt;</noscript>
  </head>
  <script src="//secure.leadforensics.com/js/79075.js"></script>
  <noscript>&lt;img src="//secure.leadforensics.com/js/79075.js" style="display:none;"&gt;</noscript>
</body>
</html>
```

They highlight fine when outside the `body` tag but not within `body` or `head`. I'm using Atom Version 1.10.2 on Mac 10.11.5

![screenshot 2016-09-19 23 40 14](https://cloud.githubusercontent.com/assets/934322/18656823/8ec75bc0-7ec2-11e6-9663-b94ff354cdac.png)
![screenshot 2016-09-19 23 41 29](https://cloud.githubusercontent.com/assets/934322/18656831/9e471202-7ec2-11e6-8b28-b3c3eb5cdfb7.png)

**Comment by simurai** [comment]
Ok, yes.. I can reproduce it inside `<body>/<head>`. Should be fixed in Atom `1.12`.
